contracts/utils/test/authorizable.ts

```diff
@@ -1,52 +1,30 @@
-import { chaiSetup, constants, provider, txDefaults, web3Wrapper } from '@0x/contracts-test-utils';
-import { BlockchainLifecycle } from '@0x/dev-utils';
+import { blockchainTests, constants, expect } from '@0x/contracts-test-utils';
 import { BigNumber } from '@0x/utils';
-import * as chai from 'chai';
 import * as _ from 'lodash';
 
 import AuthorizableRevertErrors = require('../src/authorizable_revert_errors');
 import OwnableRevertErrors = require('../src/ownable_revert_errors');
 
 import { artifacts } from './artifacts';
-import { AuthorizableContract } from './wrappers';
+import { TestAuthorizableContract } from './wrappers';
 
-chaiSetup.configure();
-const expect = chai.expect;
-const blockchainLifecycle = new BlockchainLifecycle(web3Wrapper);
-
-describe('Authorizable', () => {
+blockchainTests.resets('Authorizable', env => {
     let owner: string;
     let notOwner: string;
     let address: string;
-    let authorizable: AuthorizableContract;
-
-    before(async () => {
-        await blockchainLifecycle.startAsync();
-    });
-
-    after(async () => {
-        await blockchainLifecycle.revertAsync();
-    });
+    let authorizable: TestAuthorizableContract;
 
     before(async () => {
-        const accounts = await web3Wrapper.getAvailableAddressesAsync();
+        const accounts = await env.getAccountAddressesAsync();
```

Comment:
Nice, thanks for refactoring this file!

Reply:
No problem! It's always nice to do a refactor.